exception.not_unique_skill=Skill with title {0} already exists
exception.user_not_found=User with id {0} does not exist